server/api/internal/usecase/document/client.go

```
@@ -0,0 +1,160 @@
+func (c *Client) Rollback(ctx context.Context, id string, clock int) (*Document, error) {
+	url := fmt.Sprintf("%s/%s/rollback?clock=%d", c.baseURL, id, clock)
+	req, err := http.NewRequestWithContext(ctx, "GET", url, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
```

golangci-lint (1.62.2)

Error return value of `resp.Body.Close` is not checked

(errcheck)